content/docs/1.17/operator.md

```diff
@@ -19,29 +19,64 @@ While multiple operators might coexist watching the same set of namespaces, whic
 The Jaeger Operator version tracks one version of the Jaeger components (Query, Collector, Agent). When a new version of the Jaeger components is released, a new version of the operator will be released that understands how running instances of the previous version can be upgraded to the new version.
 {{< /info >}}
 
+## Install modes
+
+The Jaeger Operator can be installed to watch for new Jaeger resources either in the whole cluster or in specific namespaces.
```

Comment:
This needs more clarification about what you mean by "Jaeger resources".

Reply:
+1, I mean CR here and I'll change this here.